wagtailio/navigation/models.py

```diff
@@ -19,6 +21,12 @@ class FooterMenu(models.Model):
         FieldPanel("sections"),
     ]
 
+    def save(self, **kwargs):
```

Comment:
Suggestion: Perhaps this would be better handled using a signal or hook than custom `save` method?

Reply:
perhaps. debated this, and landed on the straighforward implementation here.
happy to be convinced otherwise